css/style.css

```diff
@@ -171,6 +175,8 @@ a {
   outline: 0;
   -webkit-box-shadow: 0 3px 9px rgba(0, 0, 0, 0.5);
   box-shadow: 0 3px 9px rgba(0, 0, 0, 0.5);
+  color: var(--color-text-primary);
+  color: var(--color-text-primary);
```

Comment:
I presume this is a duplicate line and one is not necessary?

Reply:
My bad. Good catch!